### PR TITLE
benchmarks: add duration limit to starter

### PR DESCRIPTION
**Pull request by falko**
## Description

Add an optional limit of the test duration.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [x] The changes are backward compatible with previous versions

Testing:
* [ ] There are unit/integration tests that verify all acceptance criteria of the issue
* [ ] New tests are written to ensure backward compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

**Comment by falko** [comment]
@pihme it currently still uses `System.exit`. Ideally, we change that to something more graceful as discussed today

**Comment by pihme** [comment]
@Zelldon don't want to make any changes here without your consent

File: benchmarks/project/src/main/java/io/zeebe/Starter.java [hunk 1]
```diff
@@ -59,30 +62,39 @@ public void run() {
...
+                        .variables(variables)
+                        .send());
+              }
+            } catch (Exception e) {
+              LOG.error("Error on creating new workflow instance", e);
             }
-          } catch (Exception e) {
-            LOG.error("Error on creating new workflow instance", e);
+          } else {
+            // TODO can one use scheduledFuture.cancel(false) to gracefully
```
Comment (Zelldon) [inline benchmarks/project/src/main/java/io/zeebe/Starter.java:94]: Who is the one ? :sweat_smile:

Comment (Zelldon) [reply benchmarks/project/src/main/java/io/zeebe/Starter.java:94]: So what you want to do is canceling the timer, stopping the execution service. Await all remaining responses and then exit right?

**Review by ChrisKujawa — DISMISSED** [review]
Like it, but could we not directly also implement the todo?

**Review by ChrisKujawa — APPROVED** [review]
Thanks guys :+1:

**Pull request merged**